### PR TITLE
Remove labels on touch devices

### PR DESCRIPTION
**Pull request by SAGV**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -165,3 +165,4 @@ pip-log.txt
 
 # Mac crap
 .DS_Store
+bower_components
```
Comment (mhogerheijde) [inline .gitignore:170]: This should not be directly under `# Mac crap`

Comment (SAGV) [inline .gitignore:170]: yeah, indeed!

File: dist/leaflet.label-src.js [hunk 2]
```diff
@@ -516,11 +516,21 @@ L.Path.include({
 });
 
 L.Map.include({
+	shownLabel: null,
+
 	showLabel: function (label) {
+
+		//Remember the last shown label to hide it on touch devices
+		if (this.shownLabel) {
```
Comment (mhogerheijde) [inline dist/leaflet.label-src.js:524]: this hides the previous label when showing a new label? Ergo: only 1 label is shown at a time?

Comment (mhogerheijde) [inline dist/leaflet.label-src.js:524]: Furthermore: do we want to PR this to the original repo? Since I guess this shouldn't be standard behaviour then.

Comment (SAGV) [inline dist/leaflet.label-src.js:524]: Labels are usually shown on `mouse over` and are hidden with `mouse out`  we don't need to show more than one label at a time

Comment (SAGV) [inline dist/leaflet.label-src.js:524]: But when we have a touch device, there is no `mouse out` event so labels stay shown all the time

**Comment by SAGV** [comment]
Fixed .gitignore

**Comment by SAGV** [comment]
@mhogerheijde can you merge it?

**Pull request merged**